### PR TITLE
Oct moc vlans update

### PR DESCRIPTION
**Pull request by naved001**

### CODE REVIEW

**Comment by hakasapl** [comment]
These docs look good to me.

**Review by hakasapl — APPROVED** [review]
Looks good to me, sorry forgot to actually click the review button.

File: docs/source/clusters/OCT-nodes.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+This rack hosts node to be used as computes for operate first cluster. The node's IPMI is on VLAN 912.
+
+Available nodes are from OCT3-00 to OCT3-31 (total of 31 nodes).
+
+Each OCT3 node has a single connection to the OCT3 10G Switch.
+
+## OCT10
+
+This rack has the newer dell hardware to be used as controlelrs for the operate first cluster. The node's IPMI is on VLAN 912.
```
Comment (larsks) [inline docs/source/clusters/OCT-nodes.md:20]: "controlelrs"

File: docs/source/clusters/OCT-nodes.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Nodes have dual 25G connections to a DellZ9100-ON. See the google sheet on top for details about switchports.
+
+## OCT4
+
+This rack has 17 nodes available to be used for ESI. These nodes' IPMI is on VLAN 911.
+
+Each node has a single 10G connection to the OCT4 10G Switch.
+
+There are also 3 intel nodes racked here, but those are not to be used under ESI. Their IPMI is also on VLAN 911.
```
Comment (larsks) [inline docs/source/clusters/OCT-nodes.md:30]: What are these nodes for, and should they be sharing the same IPMI network?

Comment (naved001) [reply docs/source/clusters/OCT-nodes.md:30]: They are used by some PhD students and some people from Intel. 

VLAN 911 is our default/general purpose IPMI network. The intel folks and the phd students get access to those 3 nodes' ipmi network via a bastion host with a firewall restricting access to only those 3 nodes.

Comment (larsks) [reply docs/source/clusters/OCT-nodes.md:30]: I was just thinking about the potential for typos and unintended consequences. If you're comfortable with the arrangement, I'm not going to complain :).

**Review by larsks — APPROVED** [review]

**Pull request merged**